geemusic/utils/music_queue.py

```diff
@@ -59,5 +62,15 @@ def reset(self, tracks=[]):
         else:
             return self.song_ids[self.current_index]
 
+    def shuffleMode(self, value, ogsong_ids=[]):
```

Comment:
Sorry about that. I've been programming in swift recently and get the two mixed up.